.github/workflows/gke.yaml

```diff
@@ -53,12 +53,12 @@ jobs:
     secrets: inherit
     with:
       hosted_provider: gke
-      rancher_version: ${{ inputs.rancher_version }}
-      k3s_version: ${{ inputs.k3s_version }}
-      operator_nightly_chart: ${{ inputs.operator_nightly_chart == false }}
-      run_p0_provisioning_tests: ${{ inputs.run_p0_provisioning_tests == true }}
-      run_p0_importing_tests: ${{ inputs.run_p0_imorting_tests == true }}
-      run_support_matrix_provisioning_tests: ${{ inputs.run_support_matrix_provisioning_tests == true }}
-      run_support_matrix_importing_tests: ${{ inputs.run_support_matrix_importing_tests == true }}
-      destroy_runner: ${{ inputs.destroy_runner == true }}
-      runner_template: ${{ inputs.runner_template }}
+      rancher_version: ${{ inputs.rancher_version || '2.8-head' }}
+      k3s_version: ${{ inputs.k3s_version || 'v1.27.9+k3s1' }}
+      operator_nightly_chart: ${{ inputs.operator_nightly_chart == false || false }}
```

Comment:
For some reason, if I use `github.event == 'schedule' && false` or `github.event == 'schedule' && true`, it always takes the value as `true`, so I have decided to use pass `false`, and same for other providers.